api/lookup.go

```diff
@@ -28,16 +29,22 @@ func LookupHandler(v *verifier.Verifier) echo.HandlerFunc {
 	return func(c echo.Context) error {
 		// Perform the unlimited verification
 		lookup, err := v.Verify(c.Param("email"))
+		deliverable := "true"
 		if err != nil {
-			return FormatEncoder(c, http.StatusInternalServerError, err)
+			if strings.Contains(err.Error(), "Fatal") {
```

Comment:
If your'e prefixing with Fatal, would a string.startsWith be faster if there's such a thing in go?